try:
    print('перед исключением')
    integer1 = int(input('введите числовое значение 1:'))
    integer2 = int(input('введите числовое значение 2:'))
    result =integer1/integer2
    result_round=round(result,2)
    print(result_round)
except ZeroDivisionError as exception1:
    print('ошибка:',exception1, ' - ' 'на "0" делить нельзя')
else:
    print('Программа работает корректно, ошибки не обнаружены')
finally:
    print('Выход из программы')
